shared/src/test/formationFactories.ts

```diff
@@ -142,7 +142,26 @@ export const generateBusinessNameAvailability = (overrides: Partial<NameAvailabi
     undefined,
   ];
   return {
-    similarNames: [`random-name-${randomInt()}`],
+    similarNames: [`some-name-${randomInt()}`],
+    status: statusValues[randomIntFromInterval("0", "5")],
```

Comment:
I don't think this is an ideal usage of `randomIntFromInterval` which was designed for zip codes. 

we have a `randomElementFromArray` fn that might suit better. You could filter out the undefined before calling it